src/ThriftSQL/ImpalaQuery.php

```diff
@@ -71,6 +71,10 @@ public function wait() {
     return $this;
   }
 
+  public function schema() {
+    throw new \RuntimeException("not implemented");
+  }
+
```

Comment:
Is this functionality possible to implement for Impala?

Reply:
Maybe I can look into it in two weeks.